rainforest/tests.go

```diff
@@ -128,6 +130,56 @@ type testElementDetails struct {
 	Elements []testElement `json:"elements,omitempty"` // for embedded tests
 }
 
+type Wisp struct {
```

Comment:
This won't solve the issue of outdated clients using an outdated schema, but why not reuse the JSON schema here rather than manually define the `Wisp` type?

Reply:
I tried a few go solutions, but either couldn't make them work or they weren't working properly. Ultimately it seemed like more of a pain(at least for this initial version) than writing it by hand.